website/src/pages/DailyPage/DailyPage.tsx

```diff
@@ -233,7 +223,19 @@ export default function DailyPage() {
     },
   ];
 
+  const {
+    dataDailySummary,
+    isLoadingDailySummary,
+    errorDailySummary,
+  } = useDailySummaryData(["OLTP", "OLTP-READONLY", "OLTP-SET", "TPCC", "TPCC_FK", "TPCC_UNSHARDED", "TPCC_FK_UNMANAGED"]);
```

Comment:
I think it would make more sense here to not hard code all the different workloads but instead rely on the default mechanism of the API route, which sends back the summary results for all supported workloads.